### ISSUE TITLE
 issue with setResourcePack for the second time.

### ISSUE THREAD

**Issue by miladm78** [MultiPaper/MultiPaper#408]
### Expected behavior

When I try to do setResourcePack a second time it works successfully

### Observed/Actual behavior

When I set the resource pack for a player for the first time, everything works correctly. However, when I try to apply the another resource pack for the player a second time, it does not happen, and I do not receive any errors

### Steps/models to reproduce

public class ResourcePackManager {

    private static final String FIRST_RESOURCE_PACK_URL = "http://test.com/a.zip";
    private static final String SECOND_RESOURCE_PACK_URL = "http://test.com/b.zip";
    private static final String HASH_KEY = "hash_key"; // Example hash key, replace with actual

    // Method to send the resource packs
    public static void sendResourcePacks(JavaPlugin plugin, Player player) {
        // Send the first resource pack
        player.setResourcePack(FIRST_RESOURCE_PACK_URL, HASH_KEY);
        
        // Schedule a task to send the second resource pack after 1 minute
        new BukkitRunnable() {
            @Override
            public void run() {
                player.setResourcePack(SECOND_RESOURCE_PACK_URL, HASH_KEY);
            }
        }.runTaskLater(plugin, 20L * 60); // 20 ticks * 60 seconds = 1 minute
    }
    
}

### Plugin and Datapack List

my ResourcePackManager

### MultiPaper version

on latest version 
[79b6271](https://github.com/MultiPaper/MultiPaper/commit/79b6271755b7208c9b21fd982836f250617139a0)

**Comment by PureGero** [comment]
I don't see any changes in multipaper's code in regards to this. Have you tested it on 1.20.1 Purpur?

**Comment by miladm78** [comment]
> I don't see any changes in multipaper's code in regards to this. Have you tested it on 1.20.1 Purpur?
Oh man, I checked on version 1.20.1 and this issue exists, but on version 1.20.4, this issue is not present and everything works fine. Also, I took the appropriate build for version 1.20.1 but still did not get any results. Is there a way to use MultiPaper on Paper 1.20.4?

**Comment by miladm78** [comment]
> I don't see any changes in multipaper's code in regards to this. Have you tested it on 1.20.1 Purpur?

Oh man, I checked on version 1.20.1 and this issue exists, but on version 1.20.4, this issue is not present and everything works fine. Also, I took the appropriate build for version 1.20.1 but still did not get any results. Is there a way to use MultiPaper on Paper 1.20.4?

**Comment by PureGero** [comment]
Nope, there is no 1.20.4 MultiPaper. If you can find the code in Paper that fixes the issue in 1.20.4, I can port it to 1.20.1 MultiPaper for you
